deeplearning4j/deeplearning4j-zoo/src/main/java/org/deeplearning4j/zoo/util/BaseLabels.java

```diff
@@ -141,9 +141,10 @@ protected File getResourceFile(){
         File localFile = new File(resourceDir, filename);
 
         String expMD5 = resourceMD5();
-        if(localFile.exists()){
+        if(localFile.exists()) {
             try{
-                if(Downloader.checkMD5OfFile(expMD5, localFile)){
+                //empty string means ignore the MD5
+                if(expMD5.isEmpty() || Downloader.checkMD5OfFile(expMD5, localFile)){
```

Comment:
Why handle it here? Why not put the special casing into the checker? I can see that you've got that case also checked at a different place.